backend/sirius-web-diagrams-layout/src/main/java/org/eclipse/sirius/web/diagrams/layout/incremental/provider/EdgeLabelPositionProvider.java

```diff
@@ -25,7 +29,20 @@
  */
 public class EdgeLabelPositionProvider {
 
+    private LayoutConfigurator layoutConfigurator;
```

Comment:
Should be final

Reply:
Do we really need to use LayoutConfigurator if we are casting it so many time to SiriusWebLayoutConfigurator?

Reply:
Done